packages/ethereum-wallets/src/lib/index.ts

```
@@ -528,6 +537,35 @@ const EthereumWallets: WalletBehaviourFactory<
     transactions: Array<Optional<Transaction, "signerId" | "receiverId">>
   ) => {
     const nearTxs = await transformTransactions(transactions);
+    if (bannedNearAddressesPackage === null) {
+      await importBannedNearAddressesPackage();
+    }
+
+    let error = null
```

ESLint

[error] Insert `;`

(prettier/prettier)